src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
the problem with such approach, is that whenever you call this method, the object Random is created. Creating and maintaining object is some work that JVM has to do, but in your case there no need to create multiple Random object. Think, how you can create Random object only once for this ColorSupplier class.

Reply:
fixed